web/app/pear/pear.module.css

```
@@ -0,0 +1,680 @@
+  left: 50%;
+  width: 128vw;
+  height: 78px;
+  color: #74b8e2;
+  opacity: 0.4;
+  overflow: visible;
+  transform: translate(-50%, -50%) rotate(2deg);
+}
+
+.waveSvg path {
+  fill: none;
+  stroke: currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)